src/covid_model_seiir_pipeline/regression/data.py

```diff
@@ -84,13 +85,12 @@ def load_covariate(self, covariate: str, location_ids: List[int]) -> pd.DataFram
         covariate_df = covariate_df.rename(columns={f'{covariate}_reference': covariate})
         return covariate_df.loc[:, index_columns + [covariate]].set_index(index_columns)
 
-
     def load_covariates(self, covariates: Iterable[str], location_ids: List[int]) -> pd.DataFrame:
         covariate_data = []
         for covariate in covariates:
             if covariate != 'intercept':
                 covariate_data.append(self.load_covariate(covariate, location_ids))
-        covariate_data = pd.concat(covariate_data, axis=1)
+        covariate_data = reduce(lambda x, y: x.merge(y, left_index=True, right_index=True), covariate_data)
```

Comment:
:partying_face:

Reply:
I was sad to resort to a merge, but pd.concat didn't like the required broadcasting/dealing with nulls.